dotnet-7.yaml

```diff
@@ -129,10 +125,21 @@ subpackages:
       runtime:
         - dotnet-7-runtime
         - dotnet-7-targeting-pack
+
 update:
   enabled: true
   github:
     identifier: dotnet/installer
     strip-prefix: v
     use-tag: true
-    tag-filter: "v7.0.1" # We specifically the 7.0.1xx releases since those are intended for prod.
```

Comment:
Was this comment stripped by the `advisory ...` command?

Reply:
No, I just fucked up the merge.  I can put it back.